wdio.conf.js

```diff
@@ -0,0 +1,329 @@
+                    'mute-audio',
+                    'disable-infobars',
+                    'autoplay-policy=no-user-gesture-required',
+                    'auto-select-desktop-capture-source=Your Entire screen'],
+            }
+        },
+        {
+            maxInstances: 1,
+            browserName: 'firefox',
+            acceptInsecureCerts: true,
+            "moz:firefoxOptions": {
+                "prefs": {
+                    "media.navigator.streams.fake": true,
+                    "media.navigator.permission.disabled": true,
+                    "media.peerconnection.ice.tcp": true,
+                    "intl.accept_languages": "en",
+                    "media.autoplay.default": 0
+                }
+            }
+        },
+        {
+            maxInstances: 1,
+            port: 4445,
+            browserName: 'safari technology preview',
+            acceptInsecureCerts: true,
+        }
+    ],
+    //
+    // ===================
+    // Test Configurations
+    // ===================
+    // Define all options that are relevant for the WebdriverIO instance here
+    //
+    // Level of logging verbosity: trace | debug | info | warn | error | silent
+    logLevel: 'info',
+    //
+    // Set specific log levels per logger
+    // loggers:
+    // - webdriver, webdriverio
+    // - @wdio/browserstack-service, @wdio/devtools-service, @wdio/sauce-service
+    // - @wdio/mocha-framework, @wdio/jasmine-framework
+    // - @wdio/local-runner
+    // - @wdio/sumologic-reporter
+    // - @wdio/cli, @wdio/config, @wdio/utils
+    // Level of logging verbosity: trace | debug | info | warn | error | silent
+    // logLevels: {
+    //     webdriver: 'info',
+    //     '@wdio/appium-service': 'info'
+    // },
+    //
+    // If you only want to run your tests until a specific amount of tests have failed use
+    // bail (default is 0 - don't bail, run all tests).
+    bail: 0,
+    //
+    // Set a base URL in order to shorten url command calls. If your `url` parameter starts
+    // with `/`, the base url gets prepended, not including the path portion of your baseUrl.
+    // If your `url` parameter starts without a scheme or `/` (like `some/path`), the base url
+    // gets prepended directly.
+    baseUrl: 'http://localhost',
+    //
+    // Default timeout for all waitFor* commands.
+    waitforTimeout: 10000,
+    //
+    // Default timeout in milliseconds for request
+    // if browser driver or grid doesn't send response
+    connectionRetryTimeout: 120000,
+    //
+    // Default request retries count
+    connectionRetryCount: 3,
+    //
+    // Test runner services
+    // Services take over a specific job you don't want to take care of. They enhance
+    // your test setup with almost no effort. Unlike plugins, they don't add new
+    // commands. Instead, they hook themselves up into the test process.
+    //services: ['chromedriver'],
+    services: ['selenium-standalone'],
```

Comment:
Is this where we could plug the grid?

Reply:
No. The configuration for the grid will look like this -> example:
    // Remote selenium grid.
    hostname: 'selenium-grid-link',
    port: 4444,
    path: '/wd/hub/',
    protocol: 'http',

Reply:
But can you pass that as command-line arguments?

Reply:
Jenkins needs to set it somehow without changing the source code.